packs/spell-effects/spell-effect-silvers-refrain.json

```
@@ -0,0 +1,56 @@
+{
+    "_id": "2W4qAipPVlapaWbc",
+    "img": "systems/pf2e/icons/spells/silvers-refrain.webp",
+    "name": "Spell Effect: Silver's Refrain",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Silver's Refrain]</p>\n<p>Weapon and unarmed attacks by allies in the area are treated as silver.</p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Silver's Refrain]</p>\n<p>Your weapon and unarmed attacks are treated as silver.</p>"
```
Don't want to imply that the effect actually works in an area, since we can't automate that part yet.